### ISSUE TITLE
关于paddleseg中的主干网络模型结构问题

### ISSUE THREAD

**Issue by yanni1995** [PaddlePaddle/PaddleSeg#1767]
请问为什么我修改了paddleseg/models/mobilenetv3.py文件中SEModule结构的HarhSigmoid函数以后，在查看导出的模型结构中还存在HardSigmoid结构，采用的模型结构是DEepLabV3P+mobilenetv3。
修改如下：
class SEModule(nn.Layer):
    def __init__(self, channel, reduction=4, name=""):
        super(SEModule, self).__init__()
        self.avg_pool = nn.AdaptiveAvgPool2D(1)
        self.conv1 = nn.Conv2D(
            in_channels=channel,
            out_channels=channel // reduction,
            kernel_size=1,
            stride=1,
            padding=0)
        self.conv2 = nn.Conv2D(
            in_channels=channel // reduction,
            out_channels=channel,
            kernel_size=1,
            stride=1,
            padding=0)

    def forward(self, inputs):
        outputs = self.avg_pool(inputs)
        outputs = self.conv1(outputs)
        outputs = F.relu(outputs)
        outputs = self.conv2(outputs)
        '''outputs = F.hardsigmoid(outputs)'''
        outputs = F.hardswish(outputs)
        return paddle.multiply(x=inputs, y=outputs)

**Comment by juncaipeng** [comment]
这个估计是没改对地方吧，你可以print出来看看是否执行了此处。

**Comment by yanni1995** [comment]
def forward(self, inputs):
        outputs = self.avg_pool(inputs)
        outputs = self.conv1(outputs)
        outputs = F.relu(outputs)
        outputs = self.conv2(outputs)
        outputs = F.sigmoid(outputs)
        print('done')
        return paddle.multiply(x=inputs, y=outputs)


运行结果：2022-02-14 14:44:29 [INFO]	[TRAIN] epoch: 1, iter: 10/80000, loss: 0.3390, lr: 0.009999, batch_cost: 0.3037, reader_cost: 0.05895, ips: 52.6877 samples/sec | ETA 06:44:51
done
done
done
done
done
done
done
done
done
done

**Comment by yanni1995** [comment]
> 这个估计是没改对地方吧，你可以打印出来看看是否执行了。

所以是修改了的，我修改的就是mobilenet中SEMoudle板块中的这个激活函数
